flink-core/src/main/java/org/apache/flink/configuration/HighAvailabilityOptions.java

```diff
@@ -220,6 +222,52 @@ public class HighAvailabilityOptions {
                                                     + "with unresolvable hostnames.")
                                     .build());
 
+    public static final ConfigOption<Map<String, String>> CLIENT_AUTHORIZATION =
```

Comment:
I would rename this one
```suggestion
    public static final ConfigOption<Map<String, String>> ZOOKEEPER_CLIENT_AUTHORIZATION =
```